releases/v0.5.0.md

```diff
@@ -0,0 +1,71 @@
+# Release Notes for v0.5.0
+Release Date: 2023-03-14
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+- Process-based isolation is now fully supported with SGX hardware added to enclave-cc CI
+- Improved validation coverage
+
```

Comment:
Generic KBS (and AS, RVPS) can be released in v0.5.0
```suggestion
+ - Generic key broker system (including [kbs](https://github.com/confidential-containers/kbs), [attestation-service](https://github.com/confidential-containers/attestation-service) and [rvps](https://github.com/confidential-containers/attestation-service/blob/main/docs/rvps.md)) is now supported.
```

Reply:
Thanks @jialez0. Could we expand the acronyms here? Something like:

- Generic Key Broker System (KBS) is now supported. This includes the [KBS](https://github.com/confidential-containers/kbs) itself, which relies on the [Attestation Service (AS)](https://github.com/confidential-containers/attestation-service) for attestation evidence verification. Reference Values are provided to the `AS` by the [Reference Value Provider Service (RVPS)](https://github.com/confidential-containers/attestation-service/blob/main/docs/rvps.md).